views/component/courseModules/template.njk

```diff
@@ -0,0 +1,27 @@
+<ul class="list module-list">
+    {% for module in modules %}
+        <li class="module-item list-item u--border--bgrey">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    <h3 class="govuk-heading-s">{{ module.title }}</h3>
+
+                    <p class="govuk-body">{{ module.description }}</p>
+
+                    {% if module.optional === true %}
+                        <p class="optional">This module is optional</p>
+                    {% endif %}
+
+                    <a href="#" class="govuk-body govuk-link">Start</a>
+                </div>
+                <div class="govuk-grid-column-one-third">
+                    <p class="govuk-body type">{{ module.type }}</p>
+                    <p class="govuk-body">{{ module.duration }}</p>
```

Comment:
We shouldn't be showing this value in its raw form. I don't know exactly how we do it in lpg-ui, but we should be converting seconds into its appropriate hours/minutes/seconds representation